sdk/eventgrid/eventgrid/test/eventGridClient.spec.ts

```diff
@@ -134,6 +136,41 @@ describe("EventGridPublisherClient", function() {
 
       assert.equal(res._response.status, 200);
     });
+
+    it("enriches events with distributed tracing information", async () => {
+      const tracer = new TestTracer();
```

Comment:
I would expect this test to also validate the appropriate `Span`s are created inside the `TestTracer`